src/components/NewMessageForm/NewMessageForm.vue

```diff
@@ -318,6 +318,8 @@ export default {
 		 */
 		rawToParsed(text) {
 			text = text.replace(/<br>/g, '\n')
+			text = text.replace(/<div>/g, '\n')
+			text = text.replace(/<\/div>/g, '')
```

Comment:
I would rather use something like (untested) `text.replace(/<div>(.*)<\/div>/g, '\n$1')` to replace the whole block rather than individual tags, which probably works anyway, but seems less correct.

Reply:
That regular expression does not work, because it eagerly matches all `</div>` until the last one. For now let's use the other replacements.